SparkPost/2web2ui
# SA-1898: Checks subscription on delete and saving reports

### What Changed

- Subscription checked on delete/saving reports

### How To Test

- Go to report builder
- Check that subscription request made on saving report
- Check that subscription request made on deleting report

### To Do
